app/views/matches/_form.html.erb

```diff
@@ -13,21 +13,34 @@
   <div class="form-group">
     <%= f.label :team_home_id, for: "team_home_id", :class => 'control-label col-lg-2' %>
     <div class="col-lg-10">
+      <% if current_user!=nil and current_user.admin==true %>
```

Comment:
Please use `nil?` or `blank?` and maybe add a helper function instead of the ugly `== true`